source/agora/test/Flash.d

```diff
@@ -374,18 +375,19 @@ public class FlashNodeFactory : TestAPIManager
             listener_address : ListenerAddress,
             registry_address : "name.registry",
             addresses_to_register : [to!string(kp.address)]};
-        return this.createFlashNode!FlashNodeImpl(kp, conf, storage);
+        return this.createFlashNode!FlashNodeImpl(kp, conf, storage, file, line);
     }
 
     /// ditto
     public RemoteAPI!TestFlashAPI createFlashNode (FlashNodeImpl = TestFlashNode)
-        (const KeyPair kp, FlashConfig conf, DatabaseStorage storage = DatabaseStorage.Local)
+        (const KeyPair kp, FlashConfig conf, DatabaseStorage storage = DatabaseStorage.Local,
+        string file = __FILE__, int line = __LINE__)
     {
         import agora.api.Handlers;
 
         RemoteAPI!TestFlashAPI api = RemoteAPI!TestFlashAPI.spawn!FlashNodeImpl(
             conf, &this.registry, this.nodes[0].address, storage,
-            10.seconds);  // timeout from main thread
+            10.seconds, 10.seconds, file, line);  // timeout from main thread
```

Comment:
Why are there two `10.seconds`?

Reply:
The first is the timeout used by the node, the second is the timeout used by the connection to the node.